content/lessons/chapter-10/updating-the-state-5.tsx

```diff
@@ -18,33 +17,43 @@ export default function UpdatingTheState5({ lang }) {
   return (
     <TransactionChallenge
       initialStack={{
-        output_0: ['SIG(YOU)'],
-        output_1: [],
+        output_0: {
+          0: ['0', 'SIG(REVOCATION_YOU_1)', 'SIG(LASZLO)', '0'],
+          1: ['SIG(YOU) 1'],
+        },
+        output_1: { 0: [] },
       }}
       answerScript={{
-        output_0: ['OP_PUSH'],
+        output_0: [
+          'OP_IF',
+          'OP_PUSH',
+          'OP_CHECKSEQUENCEVERIFY',
+          '700',
+          'PUBKEY(REVOCATION_YOU_1)',
+          'OP_ELSE',
+          'OP_CHECKMULTISIG',
+        ],
```

Comment:
I think this script has a few mistakes in it.

OUTPUTS:
```
  Amount   |   Script
-----------|---------------------------
  100000   |  OP_IF
           |    OP_PUSH 700 OP_CHECKSEQUENCEVERIFY OP_DROP
           |    OP_PUSH PUBKEY(You) OP_CHECKSIG
           |  OP_ELSE
           |    OP_2 OP_PUSH PUBKEY(revocation_you_1) OP_PUSH PUBKEY(Bob) OP_2 OP_CHECKMULTISIG
           |  OP_ENDIF
```

Reply:
This is just a `.includes()` mehtod so while not every script is present the error you were running into was actually 13 lines up where `1: ['SIG(YOU) 1]` needed to be `1: ['SIG(YOU)', '1']`